test/data/valid_openapi/custom_headers.json

```diff
@@ -0,0 +1,244 @@
+{
+	"openapi": "3.0.0",
+	"info": {
+		"description": "The bol.com API for Retailers.",
+		"version": "3.0",
+		"title": "v3"
+	},
+	"paths": {
```

Comment:
This isn't the smallest collection we can use to test the change. Also, prefer unit tests over integration tests if possible.

Reply:
The collection is still too large

Reply:
also, let's remove non-generic references in the spec
`bol.com`, `retailers`, `commissions` etc.
We can use `sample summary` etc

Reply:
@Dhroov7 ^